pkg/renamepvc/renamepvc.go

```diff
@@ -21,15 +21,16 @@ import (
 )
 
 var (
-	ErrNotBound             = errors.New("new PVC did not get bound")
-	ErrAcceptationNotYes    = errors.New("conformation was not successful please type in yes to continue")
-	ErrAcceptationUnknown   = errors.New("please type yes or no")
-	ErrVolumeAlreadyMounted = errors.New("volume already mounted")
+	ErrNotBound           = errors.New("new PVC did not get bound")
+	ErrAcceptationNotYes  = errors.New("conformation was not successful please type in yes to continue")
```

Comment:
I think that error should be renamed

Reply:
I am not sure what you mean exactly, I changed it and I also changed the naming from `Acceptation` to `Confirmation` to be align with the code.

Reply:
That's exactly it, nice one 👍🏻